src/repo_config.go

```diff
@@ -95,6 +95,32 @@ func resolveWorkingTreeRevision(vcsType string, dir string) (string, error) {
 	return strings.TrimSuffix(string(bytes.TrimSpace(out)), "+"), nil
 }
 
+func listRevisions(vcsType string, dir string) ([]string, error) {
```

Comment:
Oh. If it's intentional it's fine. I just thought you had unintentionally done it. 

Sent from my iPhone

> On Dec 31, 2014, at 19:01, Samer Masterson <notifications@github.com> wrote:
> 
> In src/repo_config.go:
> 
> > @@ -95,6 +95,32 @@ func resolveWorkingTreeRevision(vcsType string, dir string) (string, error) {
> >  	return strings.TrimSuffix(string(bytes.TrimSpace(out)), "+"), nil
> >  }
> >  
> > +func listRevisions(vcsType string, dir string) ([]string, error) {
> Haha, I had read something about Rob Pike (pbuh) preferring `(arg0 type0, arg1 type0)` to `(arg0, arg1 type0)` because of how it looks in godoc, but the standard library has many more examples of the latter :) Quinn Slack writes:
> …
> —
> Reply to this email directly or view it on GitHub.
>